cassandra-mesos-model/src/main/proto/io/mesosphere/mesos/frameworks/cassandra/model.proto

```diff
@@ -74,7 +109,11 @@ message CassandraNode {
     optional CassandraNodeTask metadataTask = 6;
```

Comment:
Why does `CassandraNode` have a `JmxConnect` I don't think it should be any higher than executor in the hierarchy and really is probably more an attribute for the `CassandraNodeTask`.